minari/storage/local.py

```diff
@@ -14,6 +18,19 @@
 __version__ = importlib.metadata.version("minari")
 
 
+def list_non_hidden_dirs(path: str) -> List[str]:
+    """List all non-hidden subdirectories."""
+    return [
+        d.name for d in os.scandir(path) if d.is_dir() and (not d.name.startswith("."))
+    ]
+
+
+def dataset_id_sort_key(dataset_id: str) -> Tuple[str, Union[str, None], str, int]:
+    """Key for sorting dataset ids first by namespace, and then alphabetically."""
+    attrs = parse_dataset_id(dataset_id)
+    return ("" if attrs[0] is None else attrs[0],) + attrs[1:]
```

Comment:
Okay, so if two datasets have the same namespace (e.g. "") and the env is None, we will get an error when try to sort, isn't it? Because sort try to compare None